samples/webhook-configure-session-parameters-enable-agent-response.js

```diff
@@ -13,40 +13,30 @@
 // limitations under the License.
 
 /**
- * Configures a webhook to enable an agent response.
+ * Webhook configures a session parameter to enable an agent response.
```

Comment:
The purpose of the sample is not quite right. This is implementing Yunchen's request for "Configure webhook to configure session parameters, which will be used to generate agent responses", correct?

If so, what he means is that the agent should just set a parameter. The response will be controlled by the agent model. So, this sample just needs to set a parameter value.

Reply:
I sent a message to Yunchen through his document to clarify.